corehq/apps/user_importer/tasks.py

```diff
@@ -34,7 +36,7 @@ def _update_progress(value, start=0):
         'errors': group_results['errors'] + user_results['errors'],
         'rows': user_results['rows']
     }
-    upload_record = UserUploadRecord.objects.get(task_id=import_users_and_groups.request.id)
+    upload_record = UserUploadRecord.objects.using(DEFAULT_DB_ALIAS).get(task_id=import_users_and_groups.request.id)
```

Comment:
why do we need to specify the default db alias?

Reply:
That was an attempt to force it to the primary, to fix the test error, which I interpreted as it reading from a not yet updated standby, which happens in a few other places in user uploads. I think that explanation wasn't actually correct for this specific test failure, but it is still worth ensuring this read comes from the primary since it is so soon after the initial write.